src/components/GradesView/GradesView.scss

```diff
@@ -1,56 +1,20 @@
 .spinner-overlay {
-    position: fixed;
-    height: 100%;
-    width: 100%;
-    top: 0;
-    left: 0;
     background-color: #999;
```

Comment:
@adamstankiewicz Did not change this as it was already present. I can update it to use css variables with some similar color from paragon.

Reply:
Got it. Let's leave it hardcoded for now then since there isn't a CSS variable to use for colors yet in `@edx/paragon` (our design tokens implementation is still a WIP, though there is an `alpha` release that exports ~1000 CSS variables now, `npm i @edx/paragon@alpha`).